pronto/utils.py

```diff
@@ -216,7 +215,7 @@ def get_oracle_goa_url() -> str:
     return current_app.config["ORACLE_GOA"]
 
 
-def connect_pg(url: Optional[str]=None):
+def connect_pg(url: str | None=None):
```

Comment:
PEP-8

```suggestion
def connect_pg(url: str | None = None):
```

Also, I believe we should not call `psycopg2` but `psycopg`.